Sources/FoundationEssentials/String/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ target_sources(FoundationEssentials PRIVATE
     String+IO.swift
     String+Internals.swift
     String+Path.swift
+    String+Bridging.swift
```

Comment:
Not sure if we care too much, but keeping this list in alphabetical order would put `String+Bridging.swift` above `String+Comparison.swift` rather than below `String+Path.swift`